docs/upgrade.html

```diff
@@ -19,6 +19,14 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_9_0" href="#upgrade_3_9_0">Upgrading to 3.9.0 from any version 0.8.x through 3.8.x</a></h4>
+    <h5><a id="upgrade_390_notable" href="#upgrade_390_notable">Notable changes in 3.9.0</a></h5>
+    <ul>
+        <li>In case you run your Kakfa clusters with no execution permission for the /tmp partition, Kafka will refuse to start due to a change in <code>zstd-jni</code> library.
+        To remediate this problem you need to pass the following environment variable to Kafka <code>ZstdTempFolder</code> pointing to a directory with execution permissions.
```

Comment:
`ZstdTempFolder` is not an environment variable. Users need to pass this JVM flag via the `KAFKA_OPTS` environment variable.